pointer.ts

```diff
@@ -69,6 +69,8 @@ export class Pointer {
     for (let i = 1, l = this.tokens.length; i < l; i++) {
       parent = value
       key = this.tokens[i]
+      if (key == '__proto__' || key == 'constructor' || key == 'prototype')
+        continue;
       // not sure if this the best way to handle non-existant paths...
       value = (parent || {})[key]
```

Comment:
`parent?.[key]`

Reply:
@ddimitrioglo - in place of the entire conditional statement?

Reply:
`value = (parent || {})[key]`

=>

`value = parent?.[key]`

Reply:
Was looking at the wrong line!

Was looking at `72` and `73`! 😝 

Will commit and push now...

Reply:
Done! 🍰